src/modules/launcher/Wox.Plugin/SharedCommands/BrowserInfo.cs

```diff
@@ -0,0 +1,215 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO.Abstractions;
+using System.Text;
+using Wox.Plugin.Logger;
+using Wox.Plugin.SharedCommands.Win32;
+
+namespace Wox.Plugin.SharedCommands
+{
+    /// <summary>
+    /// Contains information (e.g. path to executable, name...) about the default browser.
+    /// </summary>
+    public class BrowserInfo
```

Comment:
Let's rename to what it is:


```suggestion
    public class DefaultBrowserInfo
```